Makefile

```diff
@@ -31,8 +31,11 @@ test_unit:
 
 .PHONY: test_e2e_dev
 test_e2e_dev:
-	TRAINING_MACHINE_TYPE=cpu-small pytest -v -s --environment=dev --tb=short --reruns=2 tests/e2e
+	# TODO(artem) Once the dev cluster is stable, test with `cpu-small` and `--environment=dev` for `test_e2e_dev`
+	TRAINING_MACHINE_TYPE=cpu-small pytest -v -s --environment=staging --tb=short --reruns=2 tests/e2e
 
-.PHONY: test_e2e_staging
-test_e2e_staging:
+.PHONY: test_e2e_real
+test_e2e_real:
 	TRAINING_MACHINE_TYPE=gpu-small pytest -v -s --environment=staging --tb=short --reruns=2 tests/e2e
+	# TODO(artem) Once the dev cluster is stable, remove the line below:
```

Comment:
Please create an issue and put its URL here. TODO comments are difficult to remember and discover.

Reply:
Honestly, I'm still not sure if this PR is worth to be merged: it's a totally bad idea to test *all builds* against `staging` as thus we endanger stability of the staging (read: production) cluster itself. So this would be needed only as a temporary measure until tests against `dev` work fine.

Here's [the discussion](https://neuromation.slack.com/archives/CE76BT03G/p1570566939011600) (in private channel, in Russian).